server/website/tests/test_menus.py

```diff
@@ -3,7 +3,7 @@
 from website.models.menus import MenuItem, Menu, MenuItemBase
 from website.models.pages import Page
 
-from .utils import ValidationTestCase
+from tests.utils import ValidationTestCase
```

Comment:
Jag tucker att denna import bör heta from tests.utils import ValidationTestCase
```suggestion
from utils.tests import ValidationTestCase
```